runbotics/runbotics-scheduler/src/scheduler-database/credential/credential.utils.ts

```diff
@@ -23,12 +24,14 @@ interface BaseCredentialChangeMailPayload {
     collectionCreatorEmail: string;
     collectionName: string;
     credentialName: string;
+    credentialOldName?: string | undefined;
 }
 
 interface BaseCredentialNotifyMailArgs {
     executor: IUser;
     collectionId: string;
     credentialName: string;
+    credentialOldName?: string | undefined;
```

Comment:
I think you should decide if you want to require this property, and if you do, passed value can be only string or undefined, or if you want to set this parameter as optional and always require a string if this property is defined